packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -190,8 +184,8 @@ export const Demo: React.FC<ContentProps> = props => {
   const styles = useStyles();
   return (
     <div>
-      <Caption1>Examples</Caption1>
       <div className={mergeClasses(styles.root, props.className)}>
+        {/* <Title3 block>Component Examples</Title3> */}
```

Comment:
Is this left by mistake? seems like shouldn't be there.

Reply:
rem